benchmark/stubs/span.js

```diff
@@ -34,7 +34,7 @@ const span = {
     _traceFlags: {},
     _name: 'operation'
   }),
-  _startTime: 1500000000000.123456,
+  _startTime: 1500000000.123456,
```

Comment:
Why this change?

Reply:
newer eslint breaks on numbers that would be truncated when parsed.